tests/test_dryrun.py

```diff
@@ -54,3 +68,55 @@ def test_get_referenced_tables(self, tmp_path):
         assert len(response) == 1
         assert response[0]["datasetId"] == "telemetry_derived"
         assert response[0]["tableId"] == "clients_daily_v6"
+
+    def test_get_referenced_tables_dml_stripped(self):
+
+        view_file = (
+            TEST_DIR / "sql" / "moz-fx-data-shared-prod" / "telemetry" / "view.sql"
+        )
+
+        dml_stripped = DryRun(sqlfile=str(view_file), strip_dml=True)
+        dml_unstripped = DryRun(str(view_file))
+
+        assert (
```

Comment:
Also directly test `get_sql` method here. The idea is that if `get_referenced_tables()` were broken, this test wouldn't be affected.